src/handlebar-helpers.js

```diff
@@ -23,6 +23,14 @@ const helpers = {
   },
   or() {
     return Array.prototype.slice.call(arguments, 0, -1).some(Boolean);
+  },
+  split: function (data, delimiter) {
+    if (typeof data === 'string' && typeof delimiter === 'string') {
+      if (delimiter !== '') {
```

Comment:
i would actually think empty string `''` would be an ok value to pass in. with empty string, split just splits every character. Although im not sure this would make sense for your use case?

ie.
data = "apple"
delimeter = ""
data.split(delimiter);
// output = ["a","p","p","l","e"]

Reply:
Done